apps/qubit/modules/object/templates/exportSuccess.php

```diff
@@ -0,0 +1,103 @@
+
+        <div class="form-item">
+          <label><?php echo __('Type') ?></label>
+          <select name="objectType">
+            <option value="informationObject"><?php echo sfConfig::get('app_ui_label_informationobject') ?></option>
+            <option value="authorityRecord"><?php echo sfConfig::get('app_ui_label_actor') ?></option>
+            <option value="repository"><?php echo sfConfig::get('app_ui_label_repository') ?></option>
+          </select>
+        </div>
+
+        <div class="form-item">
+          <label><?php echo __('Format') ?></label>
+          <?php if ('csv' == $type): ?>
+            <select name="format">
+              <option value="xml"><?php echo __('XML') ?></option>
+              <option value="csv" selected="selected"><?php echo __('CSV') ?></option>
+            </select>
+          <?php endif; ?>
+
+          <?php if ('csv' != $type): ?>
+            <select name="format">
+              <option value="xml" selected="selected"><?php echo __('XML') ?></option>
+              <option value="csv"><?php echo __('CSV') ?></option>
+            </select>
+          <?php endif; ?>
+        </div>
+
+        <div class="form-item">
+          <div class="panel panel-default" id="exportOptions">
+            <div class="panel-body">
+              <label>
+                <input name="includeDescendants" type="checkbox"/>
+                <?php echo __('Include descendants') ?>
+              </label>
+              <label>
+                <input name="includeDrafts" type="checkbox"/>
+                <?php echo __('Include draft records') ?>
+              </label>
+              <label>
+                <input name="includeAllLevels" type="checkbox"/>
+                <?php echo __('Include all levels of description') ?>
+              </label>
+              <div class="hidden" id="exportLevels">
+                <?php echo $form->levels
+                  ->label(__('Levels of description'))
+                  ->help(__('Select the levels of description to be included in the export. If no levels are selected, the export will fail. You can use the control (Mac ⌘) and/or shift keys to multi-select values from the Levels of description menu. Descriptions that are descendants of levels not included in the export will also be excluded.'))
+                  ->renderRow(array('class' => 'form-autocomplete')) ?>
+              </div>
+              <br />
+              <?php $taxonomy = QubitTaxonomy::getById(QubitTaxonomy::LEVEL_OF_DESCRIPTION_ID) ?>
+
+            </div>
+          </div>
+        </div>
+
+
+      </fieldset>
+      </div>
+    </section>
+
+    <section class="actions">
+      <ul>
+        <li><input class="c-btn c-btn-submit" type="submit" id="exportSubmit" value="<?php echo __('Export') ?>"/></li>
+        <li><?php echo link_to(__('Cancel'), array('module' => 'user', 'action' => 'clipboard'), array('class' => 'c-btn')) ?></li>
```

Comment:
You may need to check that the "referer" exists and if it doesn't I'd redirect to the clipboard page (until this can be reached from other places)